src/Microsoft.ML.Transforms/CountFeatureSelection.cs

```diff
@@ -18,7 +18,47 @@
 
 namespace Microsoft.ML.Transforms
 {
-    /// <include file='doc.xml' path='doc/members/member[@name="CountFeatureSelection"]' />
+    /// <summary>
+    ///  Selects the slots for which the count of non-default values is greater than or equal to a threshold.
+    /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | Yes. |
+    /// | Input column data type | Vector or primitive of numeric, text or [key](xref:Microsoft.ML.Data.KeyDataViewType) data types.|
+    /// | Output column data type | Same as the input column.|
+    ///
+    /// This transform uses a set of aggregators to count the number of non-default values for each slot(vector element) and
+    /// instantiates a <xref:Microsoft.ML.Transforms.SlotsDroppingTransformer> to actually drop the slots that have the default values for the types
+    /// keeping at least 'count' non-default values.
+    /// This transform is useful when applied together with a <xref:Microsoft.ML.Transforms.OneHotHashEncodingTransformer>.
+    /// The count feature selection can remove the features generated by the hash transform that have no data in the examples.
+    ///
+    /// For example, if we set the count to 3 and fit the estimator, apply the transformer to the following Features column,
+    /// we would see the second solot, containing: NaN, 5, 5, NaN values being dropped because that slot has only two non-default values, the two 5 values.
```

Comment:
>solot [](start = 32, length = 5)

slot

please use spell checker. we won't catch all the typos manually.
[https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellChecker](https://marketplace.visualstudio.com/items?itemName=EWoodruff.VisualStudioSpellChecker)